app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,3 @@
+@import "bootstrap";
```

Comment:
not consistent quotes, this will fail `make build` as rubocop would complain?

Reply:
make build doesn't fail it. I think it is because rubocop enforces double quotes and yarn forces single quote